packages/dds/tree/src/test/simple-tree/discrepancies.spec.ts

```diff
@@ -1,832 +0,0 @@
-				true,
-			);
-			assert.equal(
-				allowsRepoSuperset(defaultSchemaPolicy, emptyLocalFieldTreeStored, emptyTreeStored),
-				true,
-			);
-
-			assert.deepEqual(
-				[
-					...getAllowedContentDiscrepancies(
-						normalizeFieldSchema(emptyTree),
-						emptyLocalFieldTreeStored,
-					),
-				],
-				[
-					{
-						fieldKey: undefined,
-						identifier: undefined,
-						mismatch: "allowedTypes",
-						stored: [],
-						view: [
-							{
-								metadata: {},
-								type: emptyTree,
-							},
-						],
-					},
-					{
-						fieldKey: undefined,
-						identifier: undefined,
-						mismatch: "fieldKind",
-						view: "Value",
-						stored: "Forbidden",
-					},
-				],
-			);
-
-			assert.deepEqual(
-				[...getAllowedContentDiscrepancies(normalizeFieldSchema(ObjectNode), emptyTreeStored)],
-				[
-					{
-						fieldKey: undefined,
-						identifier: undefined,
-						mismatch: "allowedTypes",
-						stored: [],
-						view: [
-							{
-								metadata: {},
-								type: ObjectNode,
-							},
-						],
-					},
-					{
-						fieldKey: undefined,
-						identifier: undefined,
-						mismatch: "fieldKind",
-						view: "Value",
-						stored: "Forbidden",
-					},
-					{
-						identifier: SchemaFactory.number.identifier,
-						mismatch: "nodeKind",
-						view: LeafNodeStoredSchema,
-						stored: undefined,
-					},
-					{
-						identifier: "schema discrepancies.tree",
-						mismatch: "fields",
-						differences: [
-							{
-								identifier: "schema discrepancies.tree",
-								fieldKey: "x",
-								mismatch: "fieldKind",
-								view: "Optional",
-								stored: "Forbidden",
-							},
-						],
-					},
-				],
-			);
-
-			assert.deepEqual(
-				[
-					...getAllowedContentDiscrepancies(
-						normalizeFieldSchema(ObjectNode),
-						emptyLocalFieldTreeStored,
-					),
-				],
-				[
-					{
-						fieldKey: undefined,
-						identifier: undefined,
-						mismatch: "allowedTypes",
-						stored: [],
-						view: [
-							{
-								metadata: {},
-								type: ObjectNode,
-							},
-						],
-					},
-					{
-						fieldKey: undefined,
-						identifier: undefined,
-						mismatch: "fieldKind",
-						view: "Value",
-						stored: "Forbidden",
-					},
-					{
-						identifier: SchemaFactory.number.identifier,
-						mismatch: "nodeKind",
-						view: LeafNodeStoredSchema,
-						stored: undefined,
-					},
-					{
-						identifier: "schema discrepancies.tree",
-						mismatch: "fields",
-						differences: [
-							{
-								identifier: "schema discrepancies.tree",
-								fieldKey: "x",
-								mismatch: "allowedTypes",
-								view: [{ metadata: {}, type: SchemaFactory.number }],
-								stored: [],
-							},
-							{
-								identifier: "schema discrepancies.tree",
-								fieldKey: "x",
-								mismatch: "fieldKind",
-								view: "Optional",
-								stored: "Forbidden",
-							},
-						],
-					},
-				],
-			);
-		});
-	});
-
-	describe("isViewSupersetOfStored", () => {
```

Comment:
Most of this file was just moved, though some new tests were added above. This section however only tests a function `isViewSupersetOfStored` that that was a test utility. I removed that test utility, along with its tests.